src/index.ts

```diff
@@ -3,6 +3,9 @@ import * as setup from "@extractions/setup-crate";
 
 async function main() {
   try {
+    process.env.GITHUB_TOKEN =
+      process.env.GITHUB_TOKEN || core.getInput("token");
+
     const versionSpec = core.getInput("just-version");
     const tool = await setup.checkOrInstallTool({
```

Comment:
Yesterday I published a new version of `@extractions/setup-crate` that accepts this as an argument. Example

```ts
    const tool = await setup.checkOrInstallTool(
      { owner: "casey", name: "just", versionSpec },
      { auth: githubToken }
    );
```